Src/Scripts/UnitDefinitions/Viscosity.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+    "Name": "Viscosity",
+    "BaseUnit": "Centistokes",
```

Comment:
I'm not familiar with this unit, but Centistokes struck me as maybe not the correct base unit? It seems to be "centi" of Stokes? For all the other units, we've stuck with the defined SI base unit if any, or as close to as what made sense.

Excerpt from wikipedia:
The SI unit of kinematic viscosity is m2/s. The cgs physical unit for kinematic viscosity is the stokes (St), named after George Gabriel Stokes. It is sometimes expressed in terms of centistokes (cSt).